src/content-scripts/canvas.js

```diff
@@ -57,6 +57,33 @@ function toggleCanvasOff(){
   console.log('canvas removed!');
 };
 
+var serializeOut = function() {
+  var data = ctx.toDataURL();
+  localStorage.setItem('OurCanvas', data);
+};
+
+var serializeIn = function() {
+  return storage.getItem('OurCanvas');
+};
+
+var getCopyCanvas = function() {
+  getStorage('local');
+  var img = new Image();
+  img.src = serializeIn();
+
+  $('<canvas id="graffio-canvas"></canvas>').innerHTML = ''
+  var newCanvas = $('<canvas id="graffio-canvas"></canvas>')
```

Comment:
This function adds a new canvas element and then draws on it using the data of a stored canvas element (so basically reconstructing a stored canvas)